docs/modules/ROOT/pages/references.adoc

```diff
@@ -1,98 +1,115 @@
+. [[ref:5]] Ontology (2023) Wikipedia. Wikimedia Foundation. Available at: https://en.wikipedia.org/wiki/Ontology (Accessed: January 2, 2023).
+. [[ref:6]] Barthelemy, F. et al. (2022) Core Person Vocabulary, Core person vocabulary. van Hooland, Seth - European Commission. Available at: https://semiceu.github.io/Core-Person-Vocabulary/releases/2.00/.
+. [[ref:7]] Barthelemy, F. et al. (2022) Core Public Service Vocabulary Application Profile (CPSV-AP). Available at: https://semiceu.github.io/CPSV-AP/releases/3.1.0/.
+. [[ref:8]] Dekkers , M. (2013) Asset Description Metadata Schema (ADMS), Asset description metadata schema (ADMS). W3C. Available at: https://www.w3.org/TR/vocab-adms/.
+. [[ref:9]] VOCABULARIES (2015). W3C. Available at: https://www.w3.org/standards/semanticweb/ontology#:~:text=On%20the%20Semantic%20Web%2C%20vocabularies,represent%20an%20area%20of%20concern (Accessed: January 2, 2023).
+. [[ref:10]] PwC EU Services (2017) DCAT Application Profile for data portals in Europe, Joinup. © 2015 European Union. Available at: https://joinup.ec.europa.eu/collection/semantic-interoperability-community-semic/solution/dcat-application-profile-data-portals-europe (Accessed: January 2, 2023).
+. [[ref:11]] Coyle, K. and Baker, T. (2009) Guidelines for Dublin Core™ Application Profiles. Available at: https://www.dublincore.org/specifications/dublin-core/profile-guidelines/.
+. [[ref:12]] C. Partridge, A. Mitchell, and S. de Cesare. Guidelines for developing ontological architectures in modelling and simulation. In Ontology, Epistemology, and
+Teleology for Modeling and Simulation, pages 27–57. Springer, 2013.
+. [[ref:13]] R. Soley et al. Model driven architecture. OMG white paper, 308(308):5, 2000.
+
+. [[ref:14]] J. Siegel. Object management group model driven architecture (mda) mda guide
+rev. 2.0. 2014. URL http://www.omg.org/cgi-bin/doc?ormsc/14-06-01
+. [[ref:15]] Semantic Interoperability Community. Available at: https://github.com/SEMICeu (Accessed: January 3, 2023).
+. [[ref:16]] model2owl (2021). Available at: https://github.com/OP-TED/model2owl (Accessed: January 2, 2023).
+. [[ref:17]] OSLO toolchain. Available at: https://github.com/Informatievlaanderen/OSLO-publicationenvironment-template (Accessed: January 3, 2023).
+. [[ref:18]] Costetchi, E. (2020) ePO UML architecture, section 4. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/ontology-architecture/ontology-architecture.pdf.
+. [[ref:19]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.1.2. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:20]]SEMIC Style Guide for Drafting Semantic Specifications. SEMIC. Available at: https://github.com/SEMICeu/Draft_guidelines_2020 (Accessed: January 3, 2023).
+. [[ref:21]] Miles, A. and Bechhofer, S. (eds) (2019) SKOS Simple Knowledge Organization System Reference. Available at: https://www.w3.org/TR/skos-reference/.
```

Comment:
Let's decide on the order of fields and apply consistently. Sometimes the year is at the end; sometimes, the year is after the authors.

Reply:
fixed